nixos/modules/services/networking/syncthing.nix

```diff
@@ -7,6 +7,21 @@ let
   cfg = config.services.syncthing;
   defaultUser = "syncthing";
```

Comment:
I made this change: I did set and id for syncthing but also set a username. Is this superfluous?